### PR TITLE
fix deprecation warning for layout

### PR DESCRIPTION
**Pull request by goropikari**
fix  deprecation warning for circular_layout, shell_layout and stressmajorize_layout.

### CODE REVIEW

File: src/stress.jl [hunk 1]
```diff
@@ -63,7 +63,7 @@ function stressmajorize_layout(G, p::Int=2, w=nothing, X0=randn(_nv(G), p);
 
     if w==nothing
         w = δ.^-2
-        w[!isfinite(w)] = 0
+        w[.!isfinite.(w)] = 0
```
Comment (sbromberger) [inline src/stress.jl]: Is that correct syntax? I'm not sure the leading dot should be there.

Comment (goropikari) [reply src/stress.jl]: I also see for the first time. But WARNING suggests it when we use julia v0.6.

For example
```
julia> !bitrand(3)
WARNING: !(B::BitArray) is deprecated, use .!(B) instead.
Stacktrace:
 [1] depwarn(::String, ::Symbol) at ./deprecated.jl:70
 [2] !(::BitArray{1}) at ./deprecated.jl:57
 [3] eval(::Module, ::Any) at ./boot.jl:235
 [4] eval_user_input(::Any, ::Base.REPL.REPLBackend) at ./REPL.jl:66
 [5] macro expansion at ./REPL.jl:97 [inlined]
 [6] (::Base.REPL.##1#2{Base.REPL.REPLBackend})() at ./event.jl:73
while loading no file, in expression starting on line 0
3-element BitArray{1}:
 true
 true
 true

julia> .!bitrand(3)
3-element BitArray{1}:
 false
  true
  true
```

Comment (sbromberger) [reply src/stress.jl]: Ok, weird. I tried making my own function to broadcast that way and it didn't work. This looks good. Thank you.

**Comment by sbromberger** [comment]
Thanks, @goropikari !

**Pull request merged**